tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -1,7 +1,9 @@
-import logging
-import sys
 
+from .graphql import graphql
```

Comment:
I'm putting all graphql queries in a separate file. It might be easier for further development.